<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>G3</class>
 <widget class="QWidget" name="G3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>G3</string>
  </property>
  <widget class="QRadioButton" name="rb_wcdma">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>WCDMA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_cdma2000">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>109</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CDMA2000</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_tdscdma">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>109</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TD-SCDMA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_connect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_wcdma_210">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>wcdma-mf210</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
